src/pybind/mgr/dashboard_v2/controllers/auth.py

```diff
@@ -41,6 +41,7 @@ def create(self, username, password):
             cherrypy.session.regenerate()
             cherrypy.session[Auth.SESSION_KEY] = username
             cherrypy.session[Auth.SESSION_KEY_TS] = now
+            cherrypy.session['expire_at_browser_close'] = not stay_signed_in
```

Comment:
Can you create a class attribute to store the `expire_at_browser_close` as was done with the above constants?

Reply:
Hmm, i think ``expire_at_browser_close`` is not related to the authentication, or it doesn't feel so to me. Another problem is that controllers/auth.py is importing tools.py, and ``expire_at_browser_close`` is also used in tools.py, so this is a circular dependency if i would introduce Auth.SESSION_KEY_EXPIRE_AT_BROWSER_CLOSE. That doesn't look very well.

Reply:
Relocating the SessionExpireAtBrowserCloseTool class from tools.py to controllers/auth.py is also no option in my opinion because it is not related to authentication at all.

Reply:
Any good ideas how to fix your concern?

Reply:
I do think that session / cookie handling is part of the authentication.